src/core/scalebar/qgsscalebarsettings.h

```diff
@@ -318,39 +319,64 @@ class CORE_EXPORT QgsScaleBarSettings
     /**
      * Returns the color used for lines in the scalebar.
      * \see setLineColor()
+     * \deprecated use lineSymbol() instead.
      */
-    QColor lineColor() const { return mLineColor; }
+    Q_DECL_DEPRECATED QColor lineColor() const SIP_DEPRECATED;
 
     /**
      * Sets the \a color used for lines in the scalebar.
      * \see lineColor()
+     * \deprecated use setLineSymbol() instead.
      */
-    void setLineColor( const QColor &color ) { mLineColor = color; mPen.setColor( mLineColor ); }
+    Q_DECL_DEPRECATED void setLineColor( const QColor &color ) SIP_DEPRECATED;
```

Comment:
This is used in the scalebar decorator, might want to update that now too.

Reply:
Didn't I already handle that?

Reply:
Blah you did, my bad. Lesson learned, don't review and try to make your son sleep at the same time ;)